### PR TITLE
add makentp/setupntp cases

### PR DESCRIPTION
**Pull request by tingtli**
Add/update makentp/setupntp cases 
@junxiawang @cxhong  Please help review. Thanks!

### CODE REVIEW

**Comment by cxhong** [comment]
I am fine with the test cases.  @junxiawang , can u review it?

File: xCAT-test/autotest/testcase/makentp/cases0 [hunk 1]
```diff
@@ -4,66 +4,99 @@ cmd:makentp -v
...
 end
-start:makentp
-descriptiop:makentp 
-cmd:date -s 20000101
+
+start:makentp_extntpserver_value
+descriptiop:test makentp with extntpserver is set to a specific value
+cmd:lsdef -t site -o clustersite -i extntpservers -c >/tmp/extntpserver
 check:rc==0
-cmd:if [[ "__GETNODEATTR($$CN,os)__" =~ "rhel" ]];then clock -w;else hwclock -w ; fi
```
Comment (junxiawang) [inline xCAT-test/autotest/testcase/makentp/cases0]: this line could not be deleted ,so using clock or hwclock to write the date .

File: xCAT-test/autotest/testcase/makentp/cases0 [hunk 2]
```diff
@@ -4,66 +4,99 @@ cmd:makentp -v
...
 check:rc==0
-cmd:makentp 
-check:rc==0
-check:output=~configuring management node
-cmd:date
-check:output=~$$DATE
-cmd:if [[ "__GETNODEATTR($$CN,os)__" =~ "rhel" ]];then clock ;else hwclock  ; fi
-check:output=~$$DATE
-end
-start:makentp_a
```
Comment (junxiawang) [inline xCAT-test/autotest/testcase/makentp/cases0]: makentp -a need covered in testcase,so I do not think this should be deleted

File: xCAT-test/autotest/testcase/makentp/cases0 [hunk 3]
```diff
@@ -4,66 +4,99 @@ cmd:makentp -v
...
 cmd:updatenode $$CN -P setupntp
 check:rc==0
 check:output=~Running of postscripts has completed
 check:output=~Running postscript: setupntp
+check:output=~setupntp exited with code 0
 cmd:sleep 120
 cmd:xdsh $$CN date
 check:output=~$$DATE
-cmd:if [[ "__GETNODEATTR($$CN,os)__" =~ "rhel" ]];then xdsh $$CN clock ;else xdsh $$CN hwclock  ; fi
+cmd:if [[ "__GETNODEATTR($$CN,os)__" =~ "rhel" ]];then clock ;else hwclock  ; fi
```
Comment (junxiawang) [inline xCAT-test/autotest/testcase/makentp/cases0]: I do not think this line should be modified ,for xdsh cn to change the cn's clock,not mn's clock

**Pull request merged**